cmd/event-received/handlers_test.go

```diff
@@ -49,7 +49,7 @@ func TestHandleObjectTagsAdded(t *testing.T) {
 			dynamoClient.
 				On("One", ctx, dynamo.LpaKey("123"), dynamo.DonorKey("456"), mock.Anything).
 				Return(func(ctx context.Context, pk dynamo.PK, sk dynamo.SK, v interface{}) error {
-					b, _ := json.Marshal(donordata.Provided{LpaID: "123", Tasks: donordata.Tasks{PayForLpa: task.PaymentStatePending}})
+					b, _ := json.Marshal(donordata.Provided{LpaID: "123", Tasks: task.DonorTasks{PayForLpa: task.PaymentStatePending}})
```

Comment:
does `donordata.Tasks` have to move to `task` to make progress work? I'd be more in favour of moving `task.Progress` stuff to `donordata` if that is the case